docs/source/usage/01_usage.md

```diff
@@ -5,43 +5,46 @@
-* `cifmw_use_hive`: (Bool) toggle OpenShift deployment using hive operator.
-* `cifmw_use_devscripts`: (Bool) toggle OpenShift deploying using devscripts role.
-* `cifmw_openshift_crio_stats`: (Bool) toggle collecting cri-o stats in CRC deployment.
-* `cifmw_deploy_edpm`: (Bool) toggle deploying EDPM. Default to false.
-* `cifmw_config_nmstate`: (Bool) toggle NMstate networking deployment. Default to false.
-* `cifmw_config_certmanager`: (Bool) toggle cert-manager deployment. Default to false.
-
-~~~{admonition} Words of caution
+- `cifmw_basedir`: The base directory for all of the artifacts. Defaults to
+  `~/ci-framework-data`.
+- `cifmw_crc_hostname`: Allow to set the actual CRC inventory hostname. Mostly used in the fetch_compute_facts hook.
+  in the multinode layout, especially for the reproducer case.
+- `cifmw_edpm_deploy_baremetal`: (Bool) Toggle whether to deploy edpm on compute nodes.
+  provisioned with virtual baremetal vs pre-provisioned VM.
+- `cifmw_installyamls_repos`: install_yamls repository location. Defaults to `../..`.
+- `cifmw_manifests`: Directory where k8s related manifests will be places. Defaults to
+  `{{ cifmw_basedir }}/manifests`.
+- `cifmw_path`: customized PATH. Defaults to `~/.crc/bin:~/.crc/bin/oc:~/bin:${PATH}`.
+- `cifmw_use_libvirt`: (Bool) toggle libvirt support.
+- `cifmw_use_crc`: (Bool) toggle rhol/crc usage.
+- `cifmw_use_devscripts`: (Bool) toggle devscripts usage.
+- `cifmw_openshift_kubeconfig`: (String) Path to the kubeconfig file if externally provided. If provided will be the kubeconfig to use and update after login.
+- `cifmw_openshift_api`: (String) Path to the kubeconfig file. If provided will be the API to authenticate against.
+- `cifmw_openshift_user`: (String) Login user. If provided, the user that logins.
+- `cifmw_openshift_provided_token`: (String) Initial login token. If provided, that token will be used to authenticate into OpenShift.
+- `cifmw_openshift_password`: (String) Login password. If provided is the password used for login in.
+- `cifmw_openshift_password_file`: (String) Path to a file that contains the plain login password. If provided is the password used for login in.
+- `cifmw_openshift_skip_tls_verify`: (Boolean) Skip TLS verification to login. Defaults to `false`.
+- `cifmw_use_opn`: (Bool) toggle openshift provisioner node support.
+- `cifmw_use_hive`: (Bool) toggle OpenShift deployment using hive operator.
+- `cifmw_use_devscripts`: (Bool) toggle OpenShift deploying using devscripts role.
+- `cifmw_openshift_crio_stats`: (Bool) toggle collecting cri-o stats in CRC deployment.
+- `cifmw_deploy_edpm`: (Bool) toggle deploying EDPM. Default to false.
+- `cifmw_config_nmstate`: (Bool) toggle NMstate networking deployment. Default to false.
+- `cifmw_config_certmanager`: (Bool) toggle cert-manager deployment. Default to false.
+- `cifmw_architecture_repos`: (String) Path of the architecture repository on the controller node.
```

Comment:
stupid question, why `repos` while there's only one architecture repository?

Reply:
that's indeed a right question. Because it's a typo ahahahaha :D

Let me remove that 's'